File: 2SEM_TDS/BACK-AND/Spring2/src/main/java/com/atvMVC/model/Produto.java
package com.atvMVC.model;

public class Produto {
    private int idProduto;
    private String nome;

    private double preco;
    private int quantidade;


    public Produto(){};

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getNome() {
        return nome;
    }

    public int getIdProduto() {
        return idProduto;
    }

    public double getPreco() {
        return preco;
    }

    public void setIdProduto(int idProduto) {
        this.idProduto = idProduto;
    }

    public int getQuantidade() {
        return quantidade;
    }

    public void setPreco(double preco) {
        this.preco = preco;
    }

    public void setQuantidade(int quantidade) {
        this.quantidade = quantidade;
    }
}

File: 2SEM_TDS/FRONT-END/02/Produto.js
export default class Produto{
    #nome;
    #quantidade;
    #preco;

    constructor(nome, quantidade, preco) {
        this.#nome = nome;
        this.#quantidade = quantidade;
        this.#preco = preco;
    }

    getNome(){
        return this.#nome;
    }
    getQuantidade(){
        return this.#quantidade;
    }
    getPreco(){
        return this.#preco;
    }

    setNome(nome){
        this.#nome = nome;
    }
    setQuantidade(quantidade){
        this.#quantidade = quantidade;
    }
    setPreco(preco){
        this.#preco = preco;
    }

    
}